weatherman.py

```diff
@@ -0,0 +1,80 @@
+"""
+Generating reports of temperature from given text files.
+"""
+
+import os
+import argparse
+import sys
+
+file_list = os.listdir('./weatherdata/')
+hottest_day = {}
+hottest_date = {}
+coldest_day = {}
+maxi_humidity= {}
+mini_humidity= {}
```

Comment:
All above initializations should be enclosed in a class along with related methods

Reply:
left